aten/src/ATen/native/Unique.cpp

```diff
@@ -17,43 +17,87 @@ template <typename scalar_t>
 std::tuple<Tensor, Tensor, Tensor> _unique_cpu_template(
     const Tensor& self,
     const bool sorted,
```

Comment:
having `sorted` and `sorted_input` is really confusing. Also It is really two completely different function codes and I prefer to see them separated into subfunctions.